### ISSUE TITLE
"Repositories" Content Discussion

### ISSUE THREAD

**Issue by jaredmorgs** [asciidoctor/asciidoctor.org#453]
http://asciidoctor.org/docs/user-manual/#repositories is currently available to create content for.

Use this issue as a tracker for content suggestions, or feel free to provide PRs referencing this ticket to keep track of contributions.

This content is contained in https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/user-manual.adoc

**Comment by jaredmorgs** [comment]
I think we should mention here that not all repositories are equal when it comes to Asciidoctor support. For example, GitHub is pretty great, but BitBucket (Atlassian Stash) is absolutely terrible. 

Can't speak for GitLab but I assume it is pretty good. I know Red Hat were using an implementation of GitLab to do all the docs migrations to Asciidoc.

**Comment by mojavelinux** [comment]
Definitely. Setting some expectations here would be ideal and likely save
us a lot of support questions.

**Comment by jaredmorgs** [comment]
Hot off the press for BitBucket Server installations today, is Asciidoctor
support
https://marketplace.atlassian.com/plugins/org.christiangalsterer.bitbucket.server.bitbucket-asciidoc-plugin/server/overview

This is a 3rd party plug-in that is not compatible with bitbucket.org
(Cloud). Only self-hosted server instances can be used at this stage.

There is another plugin under development, but not hosted on the
marketplace https://github.com/noamt/bitbucket-asciidoctor-addon

I would like to see both devs collaborate and unify, but that is out of
scope for this fix.

On Fri, 1 Jan 2016 at 14:07 Dan Allen <notifications@github.com> wrote:

> Definitely. Setting some expectations here would be ideal and likely save
> us a lot of support questions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/issues/453#issuecomment-168281007>
> .
>
-- 

Sent from Mobile.
